litgpt/config.py

```diff
@@ -28,7 +28,9 @@ class Config:
     n_embd: int = 4096
     rotary_percentage: float = 0.25
     parallel_residual: bool = True
-    bias: bool = True
+    attn_qkv_bias: bool = True,
```

Comment:
I suspect that the majority of models don't have `bias`, so it might be easier to revert previous behavior and specify bias as False by default.

Reply:
The only models I know that have biases are the original GPT-2 model (QKV biases, MLP biases, attn proj biases) and Grok (only QKV biases) -- we don't have them in LitGPT yet but might add them in the future

Reply:
Quick analysis shows that these models have bias:
- stablelm-base-alpha
- stablecode
- pythia
- dolly
- RedPajama-Incite
- phi

~37% of all configs have bias. So it's safe to say that by default a bias should be disabled.

----

Interestingly enough, only Phi models have a bias for lm_head.
It's disabled by default and should stay the same.